meetings/working-groups/extensions/Extension-API-docs.md

```diff
@@ -0,0 +1,150 @@
+
+- The prototypes are displayed as described in the previous section.
+- Otherwise, the format is consistent with the current format:
+  - Each overload has a separate entry in the section.
+  - The signatures indicate if the method is a specialization (`Sum(this IEnumerable<int>)`) or generic (`Where<T>(this IEnumerable<T> source, Func<T, bool> predicate)`).
+  - The section lists all extensions, from all extending types. They are sorted alphabetically, as proposed for current extension methods.
+  - The signature does not show any indication of the extending type.
+
+### Extension class page
+
+The page for the class containing extensions will need only minimal updates in how extension members are displayed.
+
+- The TOC node for the class will have additional nodes for **Properties** (including indexers), and **operators**. Classes already support this, so it should already work. Note that the node for methods displays *method groups*, not *individual overloads*. That should remain.
+- The page should also have sections for **Properties**, and **operators**.
+- The prototypes for extensions should be displayed as shown [above](#extension-member-prototypes).
+
+### Extension member page
+
+There should be a new style for extension members. This should be modeled after the existing member template, with the following changes:
+
+- The receiver type should be shown in the title and the header block.
+- The receiver parameter should have its own block. It should precede the other parameter block.
+- The prototype for the member should follow the format shown [above](#extension-member-prototypes).
+
+The emphasis on the receiver parameter reinforces the new syntax, and is necessary for disambiguation when multiple extensions declare methods with similar signatures.
+
+### Unspeakable extension type
+
+The compiler generates a nested type with an unspeakable name, and an unspeakable member, for each extension container. See the example under [implementation](https://github.com/dotnet/csharplang/blob/main/proposals/extensions.md#implementations) in the feature spec.
+
+The unspeakable nested type requires the following behavior:
+
+- The unspeakable type, for example `<>E__1<T>`, should not be displayed in the API docs.
+- The unspeakable member, for example `public static <Extension>$(IEnumerable<T> source)` should not be displayed in the API docs. However, the single parameter for this method defines the receiver type and name for all extensions in this container.
+- The public members of the unspeakable nested class provides the prototypes for all extensions. Those are important in the docs.
+
+### XML comments on the receiver parameter
+
+Developers use the `<param>` and (optionally) `<typeparam>` tags on the `extension` container to provide information about the receiver type.
```

Comment:
Does putting this on the extension container force changes to XML comments syntax? It looks weird to me to have a parameter on a type, but something similar has to happen here. I wonder if I would rather see a new element type like <receiverparam> than have an element that doesn't make sense on a class show up in IntelliSense for all classes.


You already point out that the output needs to respect this.